meson_options.txt

```diff
@@ -1,6 +1,7 @@
 option('apparmor', type: 'boolean', value: false, description: 'AppArmor support')
 option('audit', type: 'boolean', value: false, description: 'Audit support')
 option('docs', type: 'boolean', value: false, description: 'Build documentation')
+option('install-tests', type: 'boolean', value: false, description: 'Install test executables')
```

Comment:
The issue here is that the `docs` option controls whether we build the docs at all, but introducing `tests` in its current form would control only the installation of the tests since we build most of the tests by default - i.e. `-Dtests=false` would still build tests. Maybe the current behavior should change, so the tests aren't build by default, and let it be controlled by `-Dtests=`?